docs/developer_guide/deployment.md

```diff
@@ -98,6 +98,11 @@ The deployment will run 2 containers from your pre-built docker images in 1 pod,
 * yunikorn-scheduler-core (yunikorn scheduler core and shim for K8s)
 * yunikorn-scheduler-web (web UI)
 
+Alternatively, the scheduler can be deployed as a K8S scheduler plugin:
+```
+kubectl create -f deployments/scheduler/plugin.yaml
+```
+
```

Comment:
## Deploy the Scheduler

There are 2 deployment modes for yunikorn, standalone and scheduler-plugin. In standalone mode, yunikorn runs as a customized K8s scheduler and is supposed to replace the default scheduler in the cluster; in the scheduler-plugin mode, yunikorn is built based on K8s scheduling framework as a scheduler-plugin, preserves all default scheduler functionalities plus the yunikorn features. 

### Standalone
// doc to deploy as the standalone scheduler

### scheduler-plugin
// doc to deploy as a plugin

Reply:
@yangwwei , that's in here as well under the getting started page.

Reply:
Added a new comparision page that contrasts the deployment modes.

Reply:
Thanks